flink-end-to-end-tests/test-scripts/common_mesos_docker.sh

```diff
@@ -52,11 +54,19 @@ function start_flink_cluster_with_mesos() {
     set_config_key "jobmanager.rpc.address" "mesos-master"
     set_config_key "rest.address" "mesos-master"
 
-    docker exec -itd mesos-master bash -c "${FLINK_DIR}/bin/mesos-appmaster.sh -Dmesos.master=mesos-master:5050"
+    docker cp ${FLINK_DIR} mesos-master:$MESOS_FLINK_DIR
+    docker cp ${END_TO_END_DIR}/test-scripts mesos-master:$MESOS_END_TO_END_DIR
+    docker cp ${END_TO_END_DIR}/test-scripts mesos-slave:$MESOS_END_TO_END_DIR
+
+    docker exec -itd mesos-master bash -c "${MESOS_FLINK_DIR}/bin/mesos-appmaster.sh -Dmesos.master=mesos-master:5050"
     wait_rest_endpoint_up "http://${NODENAME}:8081/taskmanagers" "Dispatcher" "\{\"taskmanagers\":\[.*\]\}"
     return 0
 }
 
+function copy_logs_from_container () {
```

Comment:
Copy the logs to `$TEST_DATA_DIR/logs/` and cat them. The logs will be cleaned up automatically. Print the logs in stdout when failed will make debugging more easier.

Reply:
If we copy logs to `${FLINK_DIR}/log`, the existing mechanism will help us to clean up too. BTW, `post_test_validation` will check the error in it after the test passed or failed.

Reply:
If the logs could show up when test failed and be cleaned up when succeed, it will make sense to me.